33886 Attribute selector (#33918)
mdn/content: files/en-us/web/css/attribute_selectors/index.md

- `[attr operator value s]`
  - : Adding an `s` (or `S`) before the closing bracket causes the value to be compared case-sensitively (for characters within the {{Glossary("ASCII")}} range).

### Values

- `<attr>`

>   - : An {{cssxref("ident")}}, that is, the unquoted name of the attribute, including valid language-specific attributes (SVG, HTML, XML, etc), [`data-*` attributes](/en-US/docs/Web/HTML/Global_attributes/data-*), or author-created attributes.

Reviewer: ```suggestion
  - : An {{cssxref("ident")}}, that is, the unquoted name of the attribute. This can be any valid language-specific attribute (SVG, HTML, XML, etc), a [`data-*` attribute](/en-US/docs/Web/HTML/Global_attributes/data-*), or an author-created attribute.
```

- `<value>`

>   - : An {{cssxref("ident")}} or {{cssxref("string")}}, representing the attribute value. The value must be quotes is it contains spaces or special characters.

Reviewer: ```suggestion
  - : An {{cssxref("ident")}} or {{cssxref("string")}}, representing the attribute value. The value must be quoted if it contains spaces or special characters.
```
